src/main/java/hannah/Hannah.java

```diff
@@ -0,0 +1,137 @@
+package hannah;
+
+import hannah.command.Command;
+import hannah.task.Task;
+import hannah.task.TaskList;
+import hannah.task.ToDos;
+import hannah.task.Deadlines;
+import hannah.task.Events;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.io.*;
+
+public class Hannah {
+    private Storage storage;
+    private TaskList list;
+    private Parser parser;
+    private static final String FILE_PATH = "src/data/Hannah.txt";
+    private Ui ui;
+
+    public Hannah() {
+        this.ui = new Ui();
+        this.storage = new Storage(FILE_PATH);
+        this.parser = new Parser();
+        try {
+            this.list = storage.loadFile();
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found: " + e.getMessage());
+            this.list = new TaskList(); // Initialize to an empty list if the file isn't found
+        }
+    }
+
+    public void run() {
```

Comment:
Would it be more readable if the lines of code were separated by their functionality?